modules/ROOT/pages/design-edit-mulitple-tabs.adoc

```diff
@@ -0,0 +1,37 @@
+= Edit in Multiple Tabs in the Text Editor
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can have multiple files open at a time in API Designer's text editor.
+
+== Opening Files in New Tabs
+
+There are four ways that you can open a file in a new tab:
+
+* Open the context menu for a file in the file tree by clicking the three dots to its right, and then select *Open in new tab*.
+* Press the Command key (MacOS) or the Control key (Windows) and click the name of the file in the file tree.
+* Place your cursor in the name of a file for an `include` tag. Then, press the Command key (MacOS) or the Control key (Windows) and click the filename.
+
+[NOTE]
+====
+Clicking a file in the file tree or in the errors view at the bottom of the screen replaces the file that is open in the current tab.
+====
+
+== Navigating to Different Tabs
+
+The tab bar appears at the top of the editor only when at least two files are open.
+
+To switch to the tab to the left of the currently open one, follow either of these steps:
+* Press Option + Shift + Tab (MacOS) or <what?> (Windows).
+* Open the menu on the tab bar by clicking the ellipsis and select *Previous Tab*.
+
+To switch to the tab to the right of the currently open one, follow either of these steps:
+* Press Shift + Tab (MacOS) or <what?> (Windows).
```

Comment:
@marcoskhabie can you clarify which is the shortcut in Windows OS?

Reply:
The shortcut for this action is Alt + Tab in windows and **Option** + Tab in MacOs